Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM ubuntu:focal as app
```

Comment:
hey can you add a comment up here that says that this docker image is only being used for testing as of today's date? that way, there's no confusion later around whether or not this is used in production